core-service/src/test/java/com/crmfoodestablishment/coreservice/entity/OrderTest.java

```diff
@@ -0,0 +1,45 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrderTest {
```

Comment:
I think that this testcase should be a part of check on the service layer, not here

Reply:
This test has been removed.